src/components/filterdialog/filterdialog.js

```diff
@@ -58,13 +59,23 @@ function renderFilters(context, result, query) {
     });
 }
 
+function renderResetButton(context, query) {
+    const elem = context.querySelector('.btnResetAllFilters');
+    if (getFilterStatus(query) === true) {
+        elem.classList.remove('hide');
+        return;
+    }
+    elem.classList.add('hide');
```

Comment:
Use `toggle` when possible.
```suggestion
    elem.classList.toggle('hide', !getFilterStatus(query));
```

I'm a little wary of how this (hiding the button) will work in the TV display mode - it will probably lose focus after clicking the `Reset all filters` button.

Reply:
done in cd152f20334107bf31eb61cadf991789757142a2